packages/nitro_config/nitro_config.gemspec

```diff
@@ -1,36 +1,35 @@
 # frozen_string_literal: true
 
-$LOAD_PATH.push File.expand_path("lib", __dir__)
-
 require_relative "lib/nitro_config/version"
 
-Gem::Specification.new do |s|
-  s.name        = "nitro_config"
-  s.version     = NitroConfig::VERSION
-  s.authors     = ["Carlos Palhares", "Jill Klang"]
-  s.email       = ["chjunior@gmail.com", "jillian.emilie@gmail.com"]
-  s.homepage    = "https://github.com/powerhome/power-tools/blob/main/packages/nitro_config/docs/README.md"
-  s.summary     = "Nitro Configuration Loader"
-  s.description = "Loads Nitro configuration and makes it available to the application"
+Gem::Specification.new do |spec|
+  spec.name = "nitro_config"
+  spec.version = NitroConfig::VERSION
+  spec.authors = ["Carlos Palhares", "Jill Klang"]
+  spec.email = ["chjunior@gmail.com", "jillian.emilie@gmail.com"]
 
-  s.license = "MIT"
-  s.required_ruby_version = ">= 2.7"
+  spec.summary = "Nitro Configuration Loader"
+  spec.description = "Loads Nitro configuration and makes it available to the application"
+  spec.homepage = "https://github.com/powerhome/power-tools"
```

Comment:
this will point to power-tools and not to the specific docs for nitro_config. Are we sure that's what we want?

Reply:
@that-jill I guess so… they can navigate from there and discover other gems. This is really for rubygems.org. You reminded me that I forgot to add `audit_tracker` to our README